src/models/Factor.js

```diff
@@ -185,13 +184,7 @@ function (Okta, Q, CookieUtil, factorUtil, BaseLoginModel) {
 
     save: function () {
       var rememberDevice = !!this.get('rememberDevice');
-      var username = this.appState.get('username');
       // Set/Remove the remember device cookie based on the remember device input.
-      if (rememberDevice) {
-        CookieUtil.setDeviceCookie(username);
-      } else {
-        CookieUtil.removeDeviceCookie();
-      }
```

Comment:
I was 99% sure I did that, but clearly the 1% won this time... sigh...
I'll do that